syft/workers/node_client.py

```diff
@@ -278,5 +285,45 @@ def delete_model(self, model_id: str) -> bool:
         response = self._forward_json_to_websocket_server_worker(message)
         return self._return_bool_result(response)
 
+    async def async_fit(self, dataset_key: str, device: str = "cpu", return_ids: List[int] = None):
+        """Asynchronous call to fit function on the remote location.
+
+        Args:
+            dataset_key: Identifier of the dataset which shall be used for the training.
+            return_ids: List of return ids.
+
+        Returns:
+            See return value of the FederatedClient.fit() method.
+        """
+        if return_ids is None:
+            return_ids = [sy.ID_PROVIDER.pop()]
+
+        # Close the existing websocket connection in order to open a asynchronous connection
+        # This code is not tested with secure connections (wss protocol).
+        self.close()
+        async with websockets.connect(
```

Comment:
I've opened a PR to make this more systematic by creaing an `async_send_msg` I'm you could sue it here :) 
https://github.com/OpenMined/PySyft/pull/3454/files

Reply:
Yeah
Maybe I can call `fit` command in `async send_command` way 
I can try if it works :)